eng/pipelines/templates/jobs/live.tests.yml

```diff
@@ -144,3 +144,5 @@ jobs:
           testRunTitle: 'Live tests for ${{ parameters.ServiceDirectory }} ${{ parameters.CloudConfig.Cloud }} $(Agent.JobName)'
           ${{ if ne(parameters.TestResultsFiles, '') }}:
             testResultsFiles: ${{ parameters.TestResultsFiles }}
+
+      - template: ../steps/code-coverage.yml
```

Comment:
Communication will still fail in INT since in the pom.xml for communication sdks, jacoco is plugged into "verify" which is triggered in the task above, "'Run tests: $(Agent.JobName)'"